metrics.go

```diff
@@ -0,0 +1,34 @@
+/*
+Copyright 2017 Caicloud Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package nirvana
+
+import (
+	"github.com/caicloud/nirvana/web"
+	"github.com/prometheus/client_golang/prometheus/promhttp"
+)
+
+// Metrics installs the default prometheus metrics handler
+type Metrics struct{}
+
+func (d Metrics) Install(s web.Server, metricsPath string) {
+	if metricsPath == "" {
+		metricsPath = "/metrics"
+	}
+	if err := s.AddDescriptors(convertHandlerToDescriptor(metricsPath, promhttp.Handler())); err != nil {
```

Comment:
`convertHandlerToDescriptor` 👈  looks like we can create some syntax sugar for this?  @caitong93 @kdada

Reply:
I have discussed that with @kdada . Sugar is important, so we better have another pr for that.